src/readers/CMakeLists.txt

```diff
@@ -0,0 +1,220 @@
+# This macro generates the reader header from the F3DReaderBoilerPlate.h.in template file
+# It also adds the code in the F3DReaderInstanciator.h.in file
+macro(declare_f3d_reader)
+  cmake_parse_arguments(F3D_READER "HAS_CUSTOM_CODE" "NAME;VTK_IMPORTER;VTK_READER;DESCRIPTION" "EXTENSIONS;MIMETYPES" ${ARGN})
+
+  if(F3D_READER_HAS_CUSTOM_CODE)
+    set(F3D_READER_HAS_CUSTOM_CODE 1)
```

Comment:
this looks strange.

```
if (val)
  val = true
``` 

?

Reply:
`F3D_READER_HAS_CUSTOM_CODE` is either `TRUE` or `FALSE` with CMake, but if it is replaced in the boiler plate we need something that can be processed by `#if`

Reply:
then use two different name, eg:
F3D_READER_CUSTOM_CODE and F3D_READER_HAS_CUSTOM_CODE

Reply:
I do not see any changes here